doc/api/debugger.md

```diff
@@ -182,4 +182,16 @@ process or via URI reference to the listening debugger:
 * `node debug <URI>` - Connects to the process via the URI such as
 localhost:5858
 
+## V8 Inspector Integration for Node.js
+
+V8 Inspector integration allows attaching Chrome DevTools to Node.js
+instances for debugging and profiling.
+
+V8 Inspector can be enabled by passing `--inspect` flag when starting Node.js
```

Comment:
@TheAlphaNerd The file name I was having problems with was 155 characters long, that should not be a problem for NTFS